instances/technique/photoplethysmography.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/photoplethysmography",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "Plethysmographic determination in which the intensity of light reflected from the skin surface and the red cells below is measured to determine the blood volume of the respective area. There are two types, transmission and reflectance.",
```

Comment:
Suggestion to change this to:

"A 'photoplethysmography' is a non-invasive technique to optically detect blood volume changes in the micro-vascular bed of tissue by measuring the transmissive absorption and/or the reflection of light by/of the skin." 

@tgbugs I think the definition in InterLex should potentially be updated ?

Reply:
Def updated. Same note here, we don't need the indefinite article.